app/models/server/raw/VoipVisitor.ts

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Do we really need to create this from scratch? We do already have a whole visitor functionality, couldn't we just append what we need to that instead?

Reply:
As per the discussion here https://open.rocket.chat/group/yq4yejyrfl5qnodl9/thread/Ep9oyKmv9NGGEWXmW we will use the existing endpoints and collections for the visitor. 
But that comes with on caveat that the endpoint will contain a word "livechat". I will check if we need anything extra is needed for Voip visitor (Which seems a rare possibility) and remove the new code for the visitor.